### DISCUSSION TITLE
VST2/3 host (and wrapper)  "Elements"

### DISCUSSION THREAD

**Discussion by CTBlaze** [monocasual/giada#565] [General]
In the light of the unfortunate situation with VST2, has anybody tried Element, by Kushview? As a wrapper for VST2?
https://kushview.net/element/
Downloads of the precompiled binaries are 5$ for 14 days, BUT one can also build it, see source at github, it's under GPL.
https://github.com/kushview/Element/releases/tag/0.46.3 

@gvnnz the code could also be a good SOURCE for VSTing in Giada ;) ;) ;)

**Comment by tblrchstr** [comment]
This is super cool! Thanks for sharing

**Reply by CTBlaze** [reply]
You're welcome - please let me/us know, when you either buy or compile it.
PS: either way, you are allowed to share the binary! says their license ;)
(yes, I would buy, it it works for me, but need to try it first, esp. with Giada)

**Comment by CTBlaze** [comment]
any news, who has tried this, compiled this for 64bit Windows, and would share? ;)
